packages/teams-js/test/public/settings.spec.ts

```diff
@@ -6,6 +7,9 @@ describe('settings', () => {
   // Use to send a mock message from the app.
 
   const utils = new Utils();
+  const emptyCallback = (): void => {
```

Comment:
Why is this needed?

Reply:
We need this to send it as a parameter to `getSettings` function.

Reply:
does this not work?
```
getSettings( () => {}).toThrowError(...
```

Reply:
TypeScript doesn't allow and throws error.